aquarel/__init__.py

```diff
@@ -1,2 +1,9 @@
 from .theme import Theme
-from .utils import load_theme, list_themes
+from .utils import load_theme, list_themes, make_graph
```

Comment:
- add `aquarel.utils.make_graph` to `aquarel/__init__.py`
- Set `__all__` for wildcard `*` imports